src/modules/users/services/queries.gql.ts

```diff
@@ -6,6 +6,13 @@ export const GET_USER_BY_IDENTITY_ID = `
 			last_name
 			mail
 			accepted_tos_at
+			group {
+				permissions {
+					permission {
```

Comment:
might be good to sort the permissions by code, just to have a predictable list for testing equalTo() purposes